### PR TITLE
{math}[GCCcore/10.3.0] GLPK v5.0, GMP v6.2.1, ISL v0.23, ...

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)
depends on Autotools (new isolated PR for it yet)

### CODE REVIEW

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in total)
jrlogin11.jureca - Linux centos linux 8.3.2011, x86_64, AMD EPYC 7742 64-Core Processor, Python 3.6.8
See https://gist.github.com/9af489c6c41dd13da82fcf6193a78546 for a full test report.

**Comment by Micket** [comment]
Test report by @Micket
**FAILED**
Build succeeded for 9 out of 10 (6 easyconfigs in total)
vera-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/3ddfdfbf9c527589963b9ff47fd746d6 for a full test report.

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in total)
vera-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/1699aeb5e90347bd80e6b57bb29df358 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in total)
bear-pg0211u08b.bear.cluster - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.8.5
See https://gist.github.com/f639df44b97c404f97df0bafe7367fec for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/d86d49850ea7b9ef1e89f999da7e9381 for a full test report.

**Comment by Micket** [comment]
There was a slightly newer nettle that i missed

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in total)
vera-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/46d2347ba2ecc9119103aed8137923dc for a full test report.

**Comment by Micket** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/n/nettle/nettle-3.7.2-GCCcore-10.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+builddependencies = [
+    ('binutils', '2.36.1'),
+    ('Autotools', '20210128'),
+]
+
+dependencies = [
+    ('GMP', '6.2.1'),
+]
+
+configopts = '--disable-openssl '  # openssl is just used for the nettle-openssl example
```
Comment (branfosj) [inline easybuild/easyconfigs/n/nettle/nettle-3.7.2-GCCcore-10.3.0.eb:27]:
```suggestion
configopts = '--disable-openssl '  # openssl is just used for the nettle-openssl example and requires openssl 1.1
```

**Review by branfosj — APPROVED** [review]
lgtm

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in total)
bear-pg0211u08b.bear.cluster - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.8.5
See https://gist.github.com/29558f30835e74408c574ca801b6f717 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in total)
bber0501u03a.bb2.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) CPU E5-2690 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/0d8cd5f911a6a7297aa2b3bfa60c06a3 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/cf3aaf6810a4e67aa05abf7e2c32d88f for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @Micket!

**Pull request merged**